src/main/io/vtx_smartaudio.c

```diff
@@ -697,10 +697,17 @@ bool vtxSmartAudioInit(void)
     return true;
 }
 
+#define SA_INITPHASE_START         0
+#define SA_INITPHASE_WAIT_SETTINGS 1 // SA_CMD_GET_SETTINGS was sent and waiting for reply.
+#define SA_INITPHASE_WAIT_PITFREQ  2 // SA_FREQ_GETPIT sent and waiting for reply.
+#define SA_INITPHASE_DONE          3
+
 void vtxSAProcess(timeUs_t now)
 {
-    static char initPhase = 0;
+    static char initPhase = SA_INITPHASE_START;
+#if 0
```

Comment:
Can we have this either removed, or replaced with a name and comment, so it can be used as make argument in custom builds?